fundingmanager.go

```diff
@@ -613,6 +616,12 @@ func (f *fundingManager) handleFundingOpen(fmsg *fundingOpenMsg) {
 	fndgLog.Infof("FundingOpen: ChannelPoint(%v) with peerID(%v) is now open",
 		resCtx.reservation.FundingOutpoint, fmsg.peer.id)
 
+	// ROUTING ADDED
+	fmsg.peer.server.routingMgr.AddChannel(
+		graph.NewID(fmsg.peer.server.lightningID),
+		graph.NewID([32]byte(fmsg.peer.lightningID)),
+		&rt.ChannelInfo{float64(resCtx.reservation.OurContribution().FundingAmount + resCtx.reservation.TheirContribution().FundingAmount)},
```

Comment:
In order to be consistent with the struct initialization style throughout the rest of the project, this line should be changed to look something like: 
```go
&rt.ChannelInfo{
    Cpt: capacity, 
}
``` 

With this modified initialization it's likely that this line will be over 80 characters, therefore I recommend creating a temporary local variable above. These changes should serve to improve readability.